### DISCUSSION TITLE
Scanning Security Onion 2.3.53 with Nessus

### DISCUSSION THREAD

**Discussion by SecureOnionUser1223** [Security-Onion-Solutions/securityonion#4793] [Unsupported Versions]
Greetings and Salutations,

I've been trying to do a compliance scan with the latest version Tennable Nessus on a Security Onion machine I've built, but I get an authentication failure that mentions two errors: (1) "KEX failed: Server does not support algorithms in server_host_key_algorithms", and (2) "Unsupported key exchange algorithm : curve25519-sha256,curve25519-sha256@libssh.org,diffie-hellman-group16-sha512,diffie-hellman-group18-sha512,diffie-hellman-group14-sha256".

I've been trying all kinds of things to get the Nessus scan to work: Setting up a public/private key exchange, using "so-allow" to whitelist the IP address of the machine I have Nessus loaded on, trying to configure Wazzuh's active response to white list the IP address (which, upon reboot, removes the IP set in ossec.conf).

All of that said, for anyone that scans Security Onion 2.0 or greater with Nessus, what steps did you take to get it working exactly?

**Comment by weslambert** [comment]
You might have to alter the supported algorithms in `/etc/ssh/sshd_config`.
